EdFi.Buzz.Etl/eng/Windows/install.ps1

```diff
@@ -0,0 +1,173 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+#requires -version 5
+#requires -RunAsAdministrator
+
+[CmdletBinding()]
+param(
+  [string]
+  $InstallPath = "c:\Ed-Fi\Buzz\ETL",
+
+  [string]
+  $WinSWUrl = "https://github.com/winsw/winsw/releases/download/v2.9.0/WinSW.NETCore31.zip",
+
+  [string]
+  $DbServer = "localhost",
+
+  [int]
+  $DbPort = 5432,
+
+  [string]
+  $DbUserName = "postgres",
+
+  [string]
+  [Parameter(Mandatory = $true)]
+  $DbPassword,
+
+  [string]
+  $DbName = "edfi_buzz"
+
+)
+
+# TODO: insure this is run as an administrator
```

Comment:
Please remove - line 7 does the trick. I missed that update in one of the other files.